internal/tools/semconvkit/templates/httpconv/http.go.tmpl

```diff
@@ -54,7 +54,7 @@ var (
 		HTTPStatusCodeKey:            semconv.HTTPStatusCodeKey,
 		HTTPTargetKey:                semconv.HTTPTargetKey,
 		HTTPURLKey:                   semconv.HTTPURLKey,
-		HTTPUserAgentKey:             semconv.HTTPUserAgentKey,
+		UserAgentOriginalKey:         semconv.UserAgentOriginalKey,
```

Comment:
I think you missed to change `"go.opentelemetry.io/otel/semconv/internal/v2"` to `"go.opentelemetry.io/otel/semconv/internal/v3"` in this file.

The same comment applies to `net.go.tmpl`.

Reply:
Addressed in https://github.com/open-telemetry/opentelemetry-go/commit/4b13fdd35ea04d504939b574071eb61eeedabe68